### PR TITLE
Jormungandr: add param to force https in link

### PR DESCRIPTION
**Pull request by antoine-de**
for navitia.io to work behind reverse proxy we need to be able to force
https scheme in flask link (url_for).

To force this add the param

```
# Behind reverse proxy we might want to force https for the jormungandr
link
# default is False
FORCE_HTTPS = True
```

in the settings file.

### CODE REVIEW

File: source/jormungandr/jormungandr/index.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 # www.navitia.io
 
 from flask.ext.restful import Resource
-from flask import url_for
+from jormungandr import url_handler
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/index.py:31]: It would be easier to name it `url_for` so  we just have to change the import and not all the call.

**Comment by kinnou02** [comment]
it's good for @guikcd ?

**Comment by guikcd** [comment]
I need to test this in pre (patch seems to sucessfully apply to 0.97.0)

**Comment by guikcd** [comment]
Ok to me (thanks nginx + snakeoil certificate + patch), tested on -pre.

**Comment by l-vincent-l** [comment]
I think you can do it in a more simple way by copying: http://flask.pocoo.org/snippets/35/

**Comment by antoine-de** [comment]
not sure to understand this but seems way better especially since it seems
to allows us to be able to return http or https link depending of the
caller (we don't want https for internal call).

**Comment by antoine-de** [comment]
close, done better with #387

**Pull request closed without merging**